examples/3dsphere/baroclinic_wave.jl

```diff
@@ -138,7 +140,7 @@ function pressure(ρ, e, normuvw, z)
 end
 
 # set up 3D domain - doubly periodic box
```

Comment:
I know this has been merged already but I was going to through it recently to test visualization. What does "doubly periodic box" mean here (since this is a spherical shell)? Was this a copy/paste mistake?

Reply:
Yes, it is. I will change it next round. Thanks!